.github/workflows/slsa-builder-go.yml

```diff
@@ -151,6 +192,7 @@ jobs:
           BUILDER_BINARY: "${{ env.BUILDER_BINARY }}"
           UNTRUSTED_ENVS: "${{ inputs.env }}"
           UNTRUSTED_WORKING_DIR: "${{ inputs.working-dir }}"
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
```

Comment:
add `"` around the variable

Reply:
do we need the entire context or only some fields? Are there sensitive fields in there?

Reply:
yeah I think you're right sensitive fields should be removed. Which fields are needed? Maybe add this info the the tracking issue